### ISSUE TITLE
Change behavior so that requests and metrics are never throttled

### ISSUE THREAD

**Issue by jeffhollan** [Azure/azure-functions-host#4753]
https://docs.microsoft.com/en-us/azure/azure-monitor/app/sampling#types-of-sampling
<ExcludedTypes>Trace;Exception</ExcludedTypes>
A semi-colon delimited list of types that you do not want to be sampled. Recognized types are: Dependency, Event, Exception, PageView, Request, Trace. All instances of the specified types are transmitted; the types that are not specified are sampled.
 

The setting is here https://github.com/Microsoft/ApplicationInsights-dotnet/blob/master/src/ServerTelemetryChannel/AdaptiveSamplingTelemetryProcessor.cs#L70, not on the SamplingSettings like I was hoping. So we’ll need to figure out where this goes and what it looks like in host.json, but should be pretty easy once we decide.

**Comment by paulbatum** [comment]
I am assuming the scope of this work item to add the new settings to host.json, and then we need a separate work item with a proposal for how to make it so new apps get this new behavior (for example by updating the host.json template).

**Comment by brettsam** [comment]
@paulbatum -- were you suggeting a host.json template update b/c adding a default here is a breaking change? Ideally we'd just set `Requests;Metrics` as the default, but I'm not sure we can.

**Comment by paulbatum** [comment]
Yeah I think it would be a bit dangerous for us to just globally turn sampling off for requests for all V2 apps, it could result in some really significant changes in app insights usage for our existing customers.  So either its an update to the template for the default host.json, or we just get this new sampling behavior in V3. I'm leaning towards the former just because of how many support cases we have that are sampling related.
